.prettierrc

```
@@ -0,0 +1,4 @@
+{
+  "semi": false,
+  "printWidth": 120
+}
```

```suggestion
{
  "semi": false,
  "printWidth": 120,
  "overrides": [
    {
      "files": "*.md",
      "options": {
        "proseWrap": "always"
      }
    }
  ]
}
```
This change causes Prettier to use the `printWidth` value to enforce the wrap width.

I restricted this setting to only the Markdown, because I didn't really like what I saw happening in some YAML files when the setting was applied to them. I think it will be fine to let YAML authors use their judgement in wrapping.

It will be necessary to run the formatter if this change is made.

---

Sure! Changed and reformatted everything.